src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -601,7 +603,11 @@ class ManagedGroupRoutesSpec extends FlatSpec with Matchers with ScalatestRouteT
   }
 
   it should "fail with 404 when the group does not exist" in {
-    val samRoutes = TestSamRoutes(resourceTypes)
+    val groups = TrieMap.empty[WorkbenchGroupIdentity, WorkbenchGroup]
+    val policyDao = new MockAccessPolicyDAO(groups)
+    val samRoutes = TestSamRoutes(resourceTypes, policyAccessDAO = Some(policyDao), policies = Some(groups))
+
+    policyDao.createResource(Resource(ManagedGroupService.managedGroupTypeName, ResourceId("foo"), Set.empty)).unsafeRunSync()
```

Comment:
it's because lots of endpoints use https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SecurityDirectives.scala#L18, which calls https://github.com/broadinstitute/sam/blob/secondAuthDomain/src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala#L191...previously, without the resource which policy belongs to being existent is fine in the test, but since we added https://github.com/broadinstitute/sam/blob/secondAuthDomain/src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala#L191, we need to add the resource in tests now